src/prpy/controllers/position_command_controller.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+from . import OrController
+from ros_control_client_py import SetPositionClient, SetPositionFailed
+from ros_control_client_py.util import or_to_ros_trajectory
+
+
+class PositionCommandController(OrController):
+    """A controller for stateful position control
+
+    An interface to using the position_command_controller package
+    that is \"compatible\" with how we use OpenRAVE controllers
+    """
+    def __init__(self, namespace, controller_name, simulated=False,
+                 connection_timeout=10.0):
+        if simulated:
+            raise NotImplementedError('simulation not supported on \
+                                       PositionCommandController')
```

Comment:
This will insert a bunch of extra spaces in the middle of the string. Instead, use string concatenation.